security/src/main/resources/messages_en.properties

```diff
@@ -22,3 +22,15 @@ assign_permission_error_for_role=Unable to assign $ permission for $ role
 remove_permission_from_role_error=Unable to remove $ permission for the $ role
 remove_package_error=Unable to remove $ package from $ client
 role_remove_from_package_error=Unable to remove $ role from $ package
+client_password_policy_error=Given password is violating the policy which is defined by the client
+capital_letters_missing=Capital letters are missing in new password which is defined as per the client password policy
+small_letters_missing=Small letters are missing in new password which is defined as per the client password policy
+numbers_missing=Numbers are missing in new password which is defined as per the client password policy
+special_characters_missing =Special characters are missing in new password which is defined as per the client password policy
+spaces_missing=Spaces characters are missing in new password which is defined as per the client password policy
+regex_mismatch=Given new password is mismatch with the pattern which is defined as per the client password policy
+min_length_error=Given new password length is less than the minimum which is defined as per the client password policy
+max_length_error=Given new password length is more than the maximum which is defined as per the client password policy
+user_not_active=The user which is being trying to access is inactive
+old_new_password_match=Given new password is matching with the existing old password.Please Check.
+new_password_missing=Please provide the new password as it is required to update the password.
```

Comment:
A password is required to update the password.

Reply:
updated